src/app/buy-or-send-deso/buy-or-send-deso.component.html

```diff
@@ -31,6 +31,7 @@
           <div class="d-flex justify-content-around">
             <div class="col-10 d-flex align-items-center flex-column">
               <button
+                data-control-name="buy-or-send-buy-now-button"
```

Comment:
I won't comment on each of these, but I think everywhere we have a public key, we should log it in the interaction event.

Reply:
I think the global query param parser thing should take care of this.